### PR TITLE
Update communication-react to 1.9.0-beta.1 for chat quickstarts

### PR DESCRIPTION
**Pull request by emlynmac**
## Purpose
A new beta was released, the quickstarts need updating to this released beta

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
Dependency update

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[x] Other... Please describe:
```

### CODE REVIEW

**Review by JoshuaLai — APPROVED** [review]

File: ui-library-filesharing-chat-composite/app/package.json [hunk 1]
```diff
@@ -3,10 +3,10 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@azure/communication-calling": "1.16.3-beta.1",
+    "@azure/communication-calling": "1.17.1-beta.5",
     "@azure/communication-chat": "1.4.0-beta.1",
```
Comment (JoshuaLai) [inline ui-library-filesharing-chat-composite/app/package.json:7]: The new beta for chat is out. Not sure if we should also do this bump. https://www.npmjs.com/package/@azure/communication-chat/v/1.4.0-beta.2

Comment (emlynmac) [reply ui-library-filesharing-chat-composite/app/package.json:7]: No, as this is a breaking change and not part of the ui library beta release.

File: ui-library-starting-with-chat-stateful/package.json [hunk 2]
```diff
@@ -3,9 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@azure/communication-calling": "1.16.3-beta.1",
+    "@azure/communication-calling": "1.17.1-beta.5",
```
Comment (vhuseinova-msft) [inline ui-library-starting-with-chat-stateful/package.json:6]: As Node 16 is a recommended version for this package, peer dependencies for `@azure/communication-react` can be skipped here
https://github.blog/2020-10-13-presenting-v7-0-0-of-the-npm-cli/#

**Review by vhuseinova-msft — APPROVED** [review]

File: ui-library-starting-with-chat-stateful/package.json (hunk 2 above)
Comment (emlynmac) [reply ui-library-starting-with-chat-stateful/package.json:6]: We need to review the other dependencies and update across the board. We should be using node 18+ now (LTS version).

**Pull request merged**